app/models/message_settings.py

```diff
@@ -0,0 +1,50 @@
+from datetime import datetime
+
+from . import db
+
+
+USER_REGISTER = 'User Registration'
+USER_CONFIRM = 'User Confirmation'
+INVITE_PAPERS = 'Invitation For Papers'
+NEXT_EVENT = 'Next Event'
+NEW_SESSION = 'New Session Proposal'
+PASSWORD_RESET = 'Reset Password'
+EVENT_ROLE = 'Event Role Invitation'
+SESSION_ACCEPT_REJECT = 'Session Accept or Reject'
+SESSION_SCHEDULE = 'Session Schedule Change'
+EVENT_PUBLISH = 'Event Published'
+AFTER_EVENT = 'After Event'
+
+
+class MessageSettings(db.Model):
+    __tablename__ = 'message_settings'
+    id = db.Column(db.Integer, primary_key=True)
```

Comment:
This is the convention we are using in every table, changing will create a lot of issues right now.